features/developer_portal/payment_details.feature

```diff
@@ -0,0 +1,56 @@
+@wip
+Feature: Developer portal payment details
+
+  Background:
+    Given a provider
+    And a product "The API"
+    And the following application plan:
+      | Product | Name | default |
+      | The API | Pro  | true    |
+    And a buyer "Jane" signed up to the product
+    And the following application:
+      | Buyer | Name       | Product |
+      | Jane  | Jane's App | My API  |
+    And the buyer logs in
+
+  Rule: Backend v1
+    Background:
+      Given the product uses backend v1
+
+    Scenario: Paid plan with prepaid billing
+      Given the date is 1st January 2023
+      Given the provider has multiple applications disabled
+      And the provider is charging its buyers in prepaid mode
+      And plan "Pro" has a monthly fee of 200
+      And they go to the dev portal API access details page
+      Then I should see "Not enough credit"
```

Comment:
Hmmm... I am wondering - how does "not enough credit" appear? I can't find any references in the code.

Reply:
Ah, OK, the whole thing is `@wip`...

Reply:
copy-🍝